src/selectors/fridge.js

```diff
@@ -234,3 +234,27 @@ export const selectAverageTemperature = createSelector(
     return Number(averageTemperature).toFixed(1);
   }
 );
+
+export const selectSelectedFridgeIsInHotBreach = state => {
```

Comment:
(comment) What's the benefit of doing these selectors over just using `fridge.isInHotBreach` in UI implementation?

Reply:
We can change the implementation of how we know a fridge is in breach by changing the selector and not touching the view layer (i.e. maybe if we move to sqlite one day)